src/types/Appstream.ts

```diff
@@ -16,6 +16,7 @@ export interface Appstream {
   provides: string[]
   launchable: Launchable
   bundle: Bundle
+  metadata: Metadata
```

Comment:
Shouldn't this be metadata?: Metadata instead

Reply:
Yeah, seems better.